CMakeLists.txt

```diff
@@ -662,7 +662,7 @@ if (BUILD_PYTHON)
     if (DEFINED PYTHON_SITE_PACKAGES)
       set (OPENTURNS_PYTHON_MODULE_PATH "${PYTHON_SITE_PACKAGES}")
     else ()
-     if (WIN32)
+     if (MSVC)
```

Comment:
we're using mingw too, that's going to break our packages as on windows the install path is always "Lib/site-packages"

Reply:
Python doesn't even support MINGW, which distribution/variant are you using?
Or do you use Python modules built on MINGW with official Python, If so I don't think It's a reliable/wise choice.
In general MINGW uses the same UNIX packaging structure with some exceptions like installing shared libraries in bin directory.

Reply:
we ship mingw-built binaries for use with official cpython, that worked pretty well for us so far ;)

Reply:
Fedora: mingw64-python https://fedora.pkgs.org/35/fedora-updates-testing-x86_64/mingw64-python3-3.10.4-1.fc35.noarch.rpm.html
ArchLinux: mingw-w64-python https://archlinux.pkgs.org/rolling/jlk-x86_64/mingw-w64-python-3.7.2-1-any.pkg.tar.xz.html

> we ship mingw-built binaries for use with official cpython, that worked pretty well for us so far ;)

That's not guaranteed.

I see you cross-build them. Couldn't you observe that the mingw-python that you are using is not structured like the official MSVC one! https://github.com/openturns/openturns/blob/master/.circleci/run_docker_mingw.sh#L22 (Just to say It's not MSYS specific)

Reply:
no, we dont use mingw-python but the official python msvc binaries repackaged into archlinux